tkcalendar.py

```diff
@@ -567,6 +582,14 @@ def selection_set(self, date):
                 self._display_calendar()
                 self._display_selection()
 
+    def get_date(self):
+        return self._sel_date.strftime("%x")
+
+    def set_date(self, value):
```

Comment:
I don't see the use of this function, the  `self._textvariable.set(self._sel_date.strftime("%x"))` should be integrated in `selection_set`.